attest/certification.go

```diff
@@ -41,15 +43,12 @@ type CertificationParameters struct {
 	// Public represents the key's canonical encoding (a TPMT_PUBLIC structure).
 	// It includes the public key and signing parameters.
 	Public []byte
-	// CreateData represents the properties of a TPM 2.0 key. It is encoded
-	// as a TPMS_CREATION_DATA structure.
-	CreateData []byte
-	// CreateAttestation represents an assertion as to the details of the key.
+	// Attestation represents an assertion as to the details of the key.
 	// It is encoded as a TPMS_ATTEST structure.
-	CreateAttestation []byte
-	// CreateSignature represents a signature of the CreateAttestation structure.
+	Attestation []byte
```

Comment:
We generally avoid all breaking changes to APIs like this. This can't change unless there's a _very_ good reason to.

Reply:
Reverted. (Although I thought it is ok to clean it now when it is only in a draft release.)